src/control/agent/main.go

```diff
@@ -30,12 +30,14 @@ import (
 	"path/filepath"
 	"syscall"
 
+	"github.com/daos-stack/daos/src/control/common"
 	"github.com/daos-stack/daos/src/control/drpc"
 	"github.com/daos-stack/daos/src/control/log"
 )
 
 var (
 	runtimeDir = flag.String("runtime_dir", "/var/run/daos_agent", "The path to runtime socket directory for daos_agent")
+	logFile    = flag.String("logfile", "/tmp/daos_agent.log", "Path for the daos agent log file")
```

Comment:
This seems to answer my previous question.  So it's still a mystery why this log and the `daos.log` are not being collected by the `post { always { } }` in the `Jenkinsfile`.